config/routes.rb

```diff
@@ -18,6 +18,7 @@
     resource :account, only: %i[show update]
     resources :group_accesses, only: %i[index destroy]
     resources :groups, only: %i[index create update show] do
+      resources :accounts, only: %i(destroy), controller: 'group_accesses'
```

Comment:
```suggestion
      resources :accounts, only: %i[destroy], controller: 'group_accesses'
```

I don't understand, on my local dev it's `(destroy)` and there are parenthesis everywhere, but on here github they are all replaced by `[...]`?

Reply:
I added Rubocop and ESLint checks to the CI (and fixed aaaaaall the existing offenses) ;)

Brackets were Rubocop's default characters for array literals, I found it better than parenthesis because hey, it's an array after all :)

Reply:
great!! yes it makes sense haha